docs/docs/guides/develop/integrate.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Integrate
+titleTemplate: ":title | Develop | Tuist"
+description: Learn how to use Tuist in your CI workflows.
+---
+
+# Integrate
+
+Integration is the automated process of ensuring that changes are valid. It is often performed continuously (CI) and is a key part of the development process. Configuration is typically handled through a file in the repository, which the integration platform uses to determine how to integrate the changes.
+
+## Examples
+
+To run Tuist commands in your CI workflows, you’ll need to install it in your CI environment. The following sections provide examples of how to do this on different CI platforms.
+
+### Xcode Cloud
+
+In [Xcode Cloud](https://developer.apple.com/xcode-cloud/), which uses Xcode projects as the source of truth, you'll need to add a [post-clone](https://developer.apple.com/documentation/xcode/writing-custom-build-scripts#Create-a-custom-build-script) script to install Tuist and run the commands you need, for example `tuist generate`:
+
+:::code-group
+
+```bash [Mise]
+#!/bin/sh
+curl https://mise.jdx.dev/install.sh | sh
+mise install # Installs the version from .mise.toml
+
+# Runs the version of Tuist indicated in the .mise.toml file
+mise x tuist generate
+```
+```bash [Homebrew]
+#!/bin/sh
+brew install --formula tuist@x.y.z
+
+tuist generate
+```
+:::
+### Codemagic
+
+In [Codemagic](https://codemagic.io), you can add an additional step to your workflow to install Tuist:
+
+::: code-group
+```yaml [Mise]
+workflows:
+  lint:
+    name: Build
+    max_build_duration: 30
+    environment:
+      xcode: 15.0.1
+    scripts:
+      - name: Install Mise
+        script: |
+          curl https://mise.jdx.dev/install.sh | sh
+          mise install # Installs the version from .mise.toml
+      - name: Build
+        script: mise x tuist build
+```
+```yaml [Homebrew]
+workflows:
+  lint:
+    name: Build
+    max_build_duration: 30
+    environment:
+      xcode: 15.0.1
+    scripts:
+      - name: Install Tuist
+        script: |
+          brew install --formula tuist@x.y.z
+      - name: Build
+        script: tuist build
+```
+:::
+
+### GitHub Actions
+
+On [GitHub Actions](https://docs.github.com/en/actions) you can an additional step to install Tuist, and in the case of managing the installation of Mise, you can use the [mise-action](https://github.com/jdx/mise-action), which abstracts the installation of Mise and Tuist:
+
+::: code-group
+```yaml [Mise]
+name: test
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+jobs:
+  lint:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: jdx/mise-action@v2
+      - run: tuist build
```

Comment:
Minor nit pick, and something you'd likely want to edit yourself as there are other examples in this file -- but confusing mismatch of terms here for what is demonstrating a build but talking about linting and testing.

```suggestion
name: Build Application
on:
  pull_request:
    branches:
      - main
  push:
    branches:
      - main
jobs:
  build:
    runs-on: macos-latest
    steps:
      - uses: actions/checkout@v3
      - uses: jdx/mise-action@v2
      - run: tuist build
```